src/test/java/com/training/sample/tests/Assignment5Test.java

```diff
@@ -0,0 +1,15 @@
+package com.training.sample.tests;
+
+import com.training.sample.pages.Assignment5;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class Assignment5Test {
+    Assignment5 assignment5 = new Assignment5();
+
+    @Test
+    public void testGetString() {
+        String[] names = {"Paulo Dybala", "Federico", "Gianluigi", "Ronaldo", "Messi"};
+        Assertions.assertEquals(3, assignment5.getString(names, "Ronaldo"));
```

Comment:
instead of creating new field 'name' you can use directly
assignment5.getString(new String[]{"Paulo Dybala", "Federico", "Gianluigi", "Ronaldo", "Messi"},"Ronaldo")